tox.ini

```diff
@@ -0,0 +1,68 @@
+[tox]
+minversion = 3.14.0
+ignore_basepython_conflict = true
+
+# configures which environments run with each python version
+[testenv]
+basepython =
+    {flake8}: {env:TOXPYTHON:python3}
+    {black}: {env:TOXPYTHON:python3}
+    {py36}: {env:TOXPYTHON:python36}
+passenv = *
+
+[testenv:py36]
+skip_install = true
```

Comment:
If this repository was configured as a package, here you should use `usedevelop=true` instead of `skip_install`. Since it is not a package, the `skip_install` is okay.

Reply:
yep next step is to package it!